tests/Feature/Api/UserList/UserListApiTest.php

```diff
@@ -46,6 +46,19 @@ public function test_get_user_list()
         $this->assertEquals($userList->img_url, $data['data']['img_url']);
     }
 
+    public function test_get_user_lists()
+    {
+        $userLists = factory(UserList::class)->create();
+        $response = $this->json('GET', "/api/v1/user-lists/$userLists->user_id/lists");
+        $response->assertHeader('Content-Type', 'application/json');
+        $response->assertStatus(200);
+        $response->json($response->getContent(), true);
```

Comment:
You should just check structure of json from server, like this
$response->assertJsonStructure([ "data" => [ "*" => [ "id", "user_id", "name", "image_url" ] ] ])